src/compas_ui/objects/object.py

```diff
@@ -140,6 +143,7 @@ def state(self):
             "settings": self.settings,
             "artist": self.artist.state,
             "visible": self.visible,
+            "frame": self.frame.data,
```

Comment:
i think we can actually do `"frame": self.frame`, as long as we use `compas.json_load` and `compas.json_dump`